app/src/test/java/com/codesoom/assignment/application/AuthenticationServiceTest.java

```diff
@@ -0,0 +1,139 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.domain.User;
+import com.codesoom.assignment.domain.UserRepository;
+import com.codesoom.assignment.dto.SessionRequestData;
+import com.codesoom.assignment.errors.EmailNotFoundException;
+import com.codesoom.assignment.errors.WrongPasswordException;
+import com.codesoom.assignment.utils.JwtUtil;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.mock;
+
+class AuthenticationServiceTest {
+    private static final String SECRET = "12345678901234567890123456789010";
+    private static final String VALID_TOKEN = "eyJhbGciOiJIUzI1NiJ9.eyJ1c2VySWQiOjF9.neCsyNLzy3lQ4o2yliotWT06FwSGZagaHpKdAkjnGGw";
+    private static final String INVALID_TOKEN = "eyJhbGciOiJIUzI1NiJ9." +
+            "eyJ1c2VySWQiOjF9.ZZ3CUl0jxeLGvQ1Js5nG2Ty5qGTlqai5ubDMXZOdaD0";
```

Comment:
`INVALID_TOKEN`이 왜 유효하지 않은지 파악하기 어려운 것 같아요. 

```suggestion
    private static final String VALID_TOKEN = "eyJhbGciOiJIUzI1NiJ9.eyJ1c2VySWQiOjF9.neCsyNLzy3lQ4o2yliotWT06FwSGZagaHpKdAkjnGGw";
    private static final String INVALID_TOKEN = VALID_TOKEN + "invalid";
```

위와 같이 하는 것은 어떨까요?

Reply:
수정하였습니다. 훨씬 명확해지네요!